{"name": "# C5149 Verify if all cards and sections are present", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5149_Verify_if_all_cards_and_sections_are_present.py\", line 16, in ll_sctns_r_prsnt\n    context.app.main_page.ll_sctns_r_prsnt()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 474, in ll_sctns_r_prsnt\n    actual_text = wait.until(EC.presence_of_element_located((DT_USG_DTLS))).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1605386069624, "stop": 1605386085478}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1605386085478, "stop": 1605386088376}, {"name": "Then Check if all elements are present. 6.1. Cards: \"DEVICE ONLINE\", 6.2. \"DEVICE OFFLINE\",  6.3. \"INVENTORY\", 6.4. \"ALERT/NOTIFICATION\", 6.5. \"DATA USAGE\"", "status": "passed", "start": 1605386088377, "stop": 1605386088611}, {"name": "Then Check if sections are present. 7.1 \"Data Usage Details\", 7.2. \"Notifications/Alerts\", 7.3. \"Groups\", 7.4. \"Signal Strength\", 7.5. \"Device Location\".", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5149_Verify_if_all_cards_and_sections_are_present.py\", line 16, in ll_sctns_r_prsnt\n    context.app.main_page.ll_sctns_r_prsnt()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 474, in ll_sctns_r_prsnt\n    actual_text = wait.until(EC.presence_of_element_located((DT_USG_DTLS))).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1605386088612, "stop": 1605386106782}], "start": 1605386064288, "stop": 1605386108897, "uuid": "4dd87957-21f2-4be4-93b0-50ca79cd75bb", "historyId": "de16d6c6861a9c5a5260c30ca3ef7806", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5149 Verify if all cards and sections are present"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}